pallets/runtime/testnet/src/runtime.rs

```diff
@@ -35,6 +35,12 @@ use pallet_sto as sto;
 pub use pallet_transaction_payment::{Multiplier, RuntimeDispatchInfo, TargetedFeeAdjustment};
 use pallet_treasury as treasury;
 use pallet_utility as utility;
+
+#[cfg(feature = "testnet")]
+use pallet_testnet as testnet;
+#[cfg(not(feature = "testnet"))]
+pub use polymesh_common_utilities::empty_module as testnet;
```

Comment:
```suggestion
#[cfg(not(feature = "testnet"))]
pub use polymesh_common_utilities::empty_module as pallet_testnet;
```
(and consequence changes as needed below...)

Reply:
In this case, I'd rather to use `as testnet` because it seems more explicit, at least to me.

Reply:
To me `testnet` feels like one unnecessary indirection not adding much value over `pallet_testnet`. The latter seems more explicit if anything. I'm nixing all of these `use pallet_foo as foo;`s in my mainnet genesis refactor, as it allows more code to be shared.